conf/machine/include/rpi-base.inc

```diff
@@ -25,8 +25,15 @@ RPI_KERNEL_DEVICETREE_OVERLAYS ?= " \
     overlays/gpio-poweroff.dtbo \
     overlays/gpio-shutdown.dtbo \
     overlays/hifiberry-amp.dtbo \
+    overlays/hifiberry-amp100.dtbo \
+    overlays/hifiberry-amp3.dtbo \
+    overlays/hifiberry-amp4pro.dtbo \
     overlays/hifiberry-dac.dtbo \
     overlays/hifiberry-dacplus.dtbo \
+    overlays/hifiberry-dacplusadcpro.dtbo \
```

Comment:
This is not an actual cherry-pick. You squashed another commit into it: https://github.com/agherzan/meta-raspberrypi/commit/fc34bc3b86f9eb42296618a3ceef82f276c472f3

Could you please include both commits for backport?